docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -0,0 +1,389 @@
+sudo apt install g++ -y
+sudo apt install -y libsctp1
+sudo apt install -y libncurses5
+sudo apt install -y libwxbase3.0-0v5
+sudo apt install -y build-essential cmake
+sudo apt install -y libdbus-1-dev
+sudo apt install -y mosh vim 
+sudo apt install -y parallel
+```
+
+Then we need to unpack and install all of this stuff. Start with `libwxgtk`:
+
+```
+sudo dpkg -i libwxgtk3.0-0v5_3.0.4+dfsg-3_amd64.deb
+sudo apt update -y
+```
+
+Then onto Erlang. First `unpack` then `install`:
+
+```
+sudo dpkg -i esl-erlang_22.3.1-1~ubuntu~bionic_amd64.deb
+```
+
+```
+sudo apt-get install -f
+```
+
+Then navigate to your `miner` directory:
+
+```
+cd miner
+```
+
+And, finally, build the `miner`
+
+```
+./rebar3 as validator release
+```
+
+#### Load the Genesis Block
+
+One last step. When building from source, you'll need to manually load the genesis block once your miner is deployed and running. To do this:
+
+```
+wget https://snapshots.helium.wtf/genesis.testnet
+miner genesis load </absolute/path/to/genesis/block>
+```
+
+## Stake Tokens to Your Validator
+
+Now that your Validator node is running, the final step in the process is to formally stake TNT to your Validator. As part of the staking process the Validator address needs to both be in the staking transaction and sign the transaction. After a wallet stakes a validator node, the wallet becomes that node’s owner, has control over that validator node, and receives rewards.
+
+First, double check your wallet balance to make sure you have the `10000` TNT required:
+
+
+```
+$ helium-wallet balance
+
++-----------------------------------------------------+----------------+--------------+-----------------+
+| Address                                             | Balance        | Data Credits | Security Tokens |
++-----------------------------------------------------+----------------+--------------+-----------------+
+| 1aP7nm6mGLMFtgiHQQxbPgKcBwnuQ6ehgTgTN8zjFuxByzJ8eA5 | 10005.00000000 | 0            | 0.00000000      |
++-----------------------------------------------------+----------------+--------------+-----------------+
+```
+
+To stake tokens, we need to get the validator node address. Obtain it using the following:
+
+
+```
+miner peer addr
+```
+
+The resulting output will look like this (except with your specific validator address). The string after `/p2p/` is your Validator address:
+
+```
+/p2p/1YwLbGTCEhVbwKEehRVQRC8N3q35ydXTH1B6BQys5FB1paHssdR
+```
+
+We can now use this address with the Helium Wallet CLI `validators stake` command to formally stake the `10000` TNT required. Here's the full command using the Validator address from above as an example. (**Make sure you replace it with yours.**)
+
+```
+helium-wallet validators stake 1YwLbGTCEhVbwKEehRVQRC8N3q35ydXTH1B6BQys5FB1paHssdR 10000 --commit
+```
+
+After running this, you'll need to input your wallet passphrase to sign the transaction. 
+
+And with that, you're done. Congratulations! You're running a Helium Network Validator. 
+
```

Comment:
Suggestion:
The first thing someone new is going to ask is: "how do I know it worked?" So, maybe add how long to wait before verifying along with  some commands and links like: 
*  the validators list on the testnet-api 
*  the `in_consensus` cmdline
*  the `info summary` cmdline

Reply:
Yup. I'll add a section on a few things to test once you've deployed and staked.

Reply:
Will do this today.